### PR TITLE
created delete and edit user component

### PR DESCRIPTION
**Pull request by DwinaTech**

### CODE REVIEW

File: frontend/src/Components/Users/UsersListTable.js [hunk 1]
```diff
@@ -109,19 +118,77 @@ export default class UsersListTable extends Component {
         <TableBody>
           {data
             .slice(page * rowsPerPage, page * rowsPerPage + rowsPerPage)
-            .map(n => (
-              <tr key={n.id}>
-                <TableCell className="user-text">{n.name}</TableCell>
-                <TableCell className="user-text">{n.email}</TableCell>
-                <Hidden xsDown>
-                  <TableCell className="user-text">{n.role}</TableCell>
+            .map((n, d) => {
```
Comment (kkarimi) [inline frontend/src/Components/Users/UsersListTable.js]: `n` and `d` are not good variable names, please use explicit naming

File: frontend/src/Components/Users/UsersListTable.js [hunk 2]
```diff
@@ -91,10 +76,34 @@ export default class UsersListTable extends Component {
     this.setState({ rowsPerPage: event.target.value });
   };
 
+  removeUser = (i) => {
+    this.setState(state => ({
+      data: state.data.filter((row, j) => j !== i)
```
Comment (kkarimi) [inline frontend/src/Components/Users/UsersListTable.js]: Is J...index?

File: frontend/src/Components/Users/UsersListTable.js [hunk 3]
```diff
@@ -91,10 +76,34 @@ export default class UsersListTable extends Component {
...
+    }))
+  }
+
+  startEditing = i => {
```
Comment (kkarimi) [inline frontend/src/Components/Users/UsersListTable.js]: is i.. index?

File: frontend/src/Components/Users/UsersListTable.js [hunk 4]
```diff
@@ -91,10 +76,34 @@ export default class UsersListTable extends Component {
...
+    this.setState({ editIdx: i });
+  };
+
+  stopEditing = () => {
+    this.setState({ editIdx: -1 });
+  };
+
+  handleChange = (e, i) => {
```
Comment (kkarimi) [inline frontend/src/Components/Users/UsersListTable.js]: `handleChange` is not explicit, `handleUserDataChange`?

File: frontend/src/Components/Users/UsersListTable.js [hunk 5]
```diff
@@ -109,19 +118,77 @@ export default class UsersListTable extends Component {
...
+                        </Select>
+                      </FormControl>
+
+                    </TableCell>
+                    <TableCell className="user-text">
+                      <Button
+                        variant="raised"
+                        type="submit"
+                        className="edit-user-button"
+                        onClick={() => this.stopEditing()}
```
Comment (kkarimi) [inline frontend/src/Components/Users/UsersListTable.js]: why not just `{this.stopEditing()}`?

File: frontend/src/Components/Users/usersData.js [hunk 6]
```diff
@@ -0,0 +1,17 @@
+const usersData = [
```
Comment (kkarimi) [inline frontend/src/Components/Users/usersData.js]: this is just a JSON file, not sure why it was made to a `.js` file

File: frontend/src/Components/Users/UsersListTable.js (hunk 1 above)
Comment (DwinaTech) [reply frontend/src/Components/Users/UsersListTable.js]: changed to better name

File: frontend/src/Components/Users/UsersListTable.js (hunk 2 above)
Comment (DwinaTech) [reply frontend/src/Components/Users/UsersListTable.js]: changed to better name

File: frontend/src/Components/Users/UsersListTable.js (hunk 3 above)
Comment (DwinaTech) [reply frontend/src/Components/Users/UsersListTable.js]: changed to better name

File: frontend/src/Components/Users/UsersListTable.js (hunk 4 above)
Comment (DwinaTech) [reply frontend/src/Components/Users/UsersListTable.js]: done

File: frontend/src/Components/Users/UsersListTable.js (hunk 5 above)
Comment (DwinaTech) [reply frontend/src/Components/Users/UsersListTable.js]: corrected

File: frontend/src/Components/Users/usersData.js (hunk 6 above)
Comment (DwinaTech) [reply frontend/src/Components/Users/usersData.js]: Changed to JSON file

**Pull request merged**